src/applications/yellow-ribbon/containers/SearchForm/index.jsx

```diff
@@ -232,7 +232,7 @@ export class SearchForm extends Component {
         {/* Unlimited Contribution Amount */}
         <ErrorableCheckbox
           checked={contributionAmount === 'unlimited'}
-          label="Only show schools that fund all tuition and fees not covered by Post-9/11 GI Bill Benefits"
+          label="Only show schools that fund all tuition and fees not covered by Post-9/11 GI Bill benefits"
```

Comment:
This is an interesting change - benefits 🤔 _benefits_.. .__b__ enefits? How does this _benefit_ the scalability of the website?

Reply:
hahaha TROLLING!